wolfcrypt/src/asn.c

```diff
@@ -15964,6 +15964,12 @@ int DecodeECC_DSA_Sig_Bin(const byte* sig, word32 sigLen, byte* r, word32* rLen,
     if (s)
         XMEMCPY(s, (byte*)sig + idx, len);
 
+#ifndef NO_STRICT_ECDSA_LEN
```

Comment:
This could also use a comment like `make sure that parsed ints exhaust the input`.

Reply:
good suggestion, adding in the comments

Reply:
Thanks!